packages/ui-utils/src/index.ts

```diff
@@ -21,9 +21,6 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
  * SOFTWARE.
  */
-export { Browser } from './Browser'
-export { isEdge } from './isEdge'
-export { isIE11 } from './isIE11'
```

Comment:
Removing Browser and isIE11 would be a breaking change. Please add them back and add to its documentation that its deprecated (use @deprecated) and will be removed in v9